pre_commit/output.py

```diff
@@ -1,6 +1,10 @@
 from __future__ import unicode_literals
 
 import sys
+try:
+    from contextlib import ExitStack
+except ImportError:
+    from contextlib2 import ExitStack
```

Comment:
when possible, prefer putting conditional imports after all the other imports (the import reorderer stops when it sees non-import code)